### DISCUSSION TITLE
Creating a new "Renovate and changelogs" page in the docs

### DISCUSSION THREAD

**Discussion by HonkingGoose** [renovatebot/renovate#25867] [Suggest an Idea]
### Background

Many users ask about how Renovate fetches changelogs. The maintainers of course answer most of those questions, but we should get those answers and concepts into the docs.

### Current info

The Renovate docs mention changelogs in a few places:

- [Private packages, looking up changelogs](https://docs.renovatebot.com/getting-started/private-packages/#looking-up-changelogs)
- [Maven datasource, making your changelogs fetchable](https://docs.renovatebot.com/modules/datasource/maven/#making-your-changelogs-fetchable)
- [Github.com token for changelogs](https://docs.renovatebot.com/getting-started/running/#githubcom-token-for-changelogs)
- [How to get Renovate to see Docker image changelogs, as image maintainer](https://docs.renovatebot.com/modules/datasource/docker/)
- [`fetchChangeLogs` config option](https://docs.renovatebot.com/configuration-options/#fetchchangelogs)
- [`customChangelogUrl` config option](https://docs.renovatebot.com/configuration-options/#customchangelogurl)
- Some Renovate managers have a "Renovate fetches changelogs step" in the how it works sections, but most do _not_ mention how to get changelogs visible to Renovate

### Related issue:

- #14138

### Discussion

It would be nice to have a single page that explains common questions and concepts, like:

- How Renovate "sees" changelogs
- From which platforms can Renovate fetch changelogs
- Troubleshooting missing changelogs
- Advice for package maintainers (do `x` to get Renovate to see changelogs)
  - npm package advice
  - yarn package advice
  - other "big/much used" package advice
- Apparently Renovate can read a `changelog.json` file, again JSON, I don't think we explain that at all...
- Other things I'm forgetting to list, feel free to suggest!

**Comment by rarkins** [comment]
Converted to docs issue: #25874
